componentens/Landingpage/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { use } from "react";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function Landingpage() {
+  const [entry, setEntry] = useState("");
+  const [message, setMessage] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    setMessage(`${entry}`);
+    setEntry("");
+  };
+  return (
+    <>
+      <container>
+        {" "}
+        <Form onSubmit={handleSubmit}>
+          <p>Where is the journey going?</p>
+          <Label htmlFor="name">City or Country?</Label>
+          <input
+            type="text"
+            required
+            id="entry"
```

Comment:
The `htmlFor` attribute is connected to the `id` of the input. You have to use the same value for both in order for the input to be accessible.